apps/emqx_dashboard/src/emqx_dashboard_audit.erl

```diff
@@ -19,30 +19,52 @@
 -include_lib("emqx/include/logger.hrl").
 -include_lib("emqx/include/http_api.hrl").
 %% API
--export([log/2]).
-
-%% filter high frequency events
--define(HIGH_FREQUENCY_REQUESTS, [
-    <<"/publish">>,
-    <<"/clients/:clientid/subscribe">>,
-    <<"/clients/:clientid/unsubscribe">>,
-    <<"/publish/bulk">>,
-    <<"/clients/:clientid/unsubscribe/bulk">>,
-    <<"/clients/:clientid/subscribe/bulk">>,
-    <<"/clients/kickout/bulk">>
-]).
-
-log(#{code := Code, method := Method} = Meta, Req) ->
+-export([log/2, log_fun/0, importance/1]).
+
+%% In the previous versions,
+%% this module used the request method to determine whether the request should be logged,
+%% but here are some exceptions:
+%% 1. the OIDC callback uses the `GET` method, but it is important
+%% 2. some endpoints (called frequency requests) use the `POST` method,
+%%    but most of the time we do not want to log them
+%% So an auxiliary  `importance` metadata was introduced.
+%%
+%% The strategy is:
+%% 1. Use `high` to mark an important `GET` method
+%% 2. Use `low` to mark the frequency methods
+%% 3. `medium` is the default importance and is set automatically
+%%
+%% Q: requests that are not allowed, e.g. a 'GET' to a 'POST'-only endpoint will not be logged, should it be a logger?
+
+-define(AUDIT_IMPORTANCE_HIGH, 100).
+-define(AUDIT_IMPORTANCE_MEDIUM, 60).
+-define(AUDIT_IMPORTANCE_LOW, 30).
```

Comment:
Maybe easier when troubleshooting the metadata at run time when it's atom used.
Recommend to resolve atom to number only when compare is needed:
```
level_to_num(high) -> 100;
level_to_num(medium) -> 60;
level_to_num(low) -> 30.
is_not_higher_level(L1, L2) -> level_to_num(L1) =< level_to_num(L2).
```